src/include/hip/rpp_hip_common.hpp

```diff
@@ -1410,6 +1410,38 @@ __device__ __forceinline__ void rpp_hip_math_subtract24_const(d_float24 *src_f24
     dst_f24->f4[5] = src_f24->f4[5] - subtrahend_f4;
 }
 
+// /******************** DEVICE RANDOMIZATION HELPER FUNCTIONS ********************/
+
+__device__ __forceinline__ float rpp_hip_rng_xorwow_f32(RpptXorwowState *xorwowState)
+{
+    uint t  = xorwowState->x[4];
```

Comment:
need to rewrite this function. Not sure why x[1]..x[4] are all the same

Reply:
same as https://github.com/GPUOpen-ProfessionalCompute-Libraries/rpp/pull/106#discussion_r862036992